build_web_compilers/lib/src/common.dart

```diff
@@ -57,3 +57,22 @@ void validateOptions(Map<String, dynamic> config, List<String> supportedOptions,
         'longer have any effect and should be removed.');
   }
 }
+
+/// Copied to `web/stack_trace_mapper.dart`, these need to be kept in sync.
```

Comment:
[nit] add a header sentence before the detail about how it is copied. Maybe the next sentence could serve as the header, it could be a bit shorter too.

```dart
/// Fixes up the [uris] from a source map so they make sense in a browser context.
```